cmd/minishift/cmd/start.go

```diff
@@ -411,8 +412,14 @@ func exportContainerImages(hostVm *host.Host, version string) {
 
 // calculateDiskSizeInMB converts a human specified disk size like "1000MB" or "1GB" and converts it into Megabits
 func calculateDiskSizeInMB(humanReadableDiskSize string) int {
+	// pre-process human readable string
+	if stringUtils.HasOnlyNumbers(humanReadableDiskSize) {
+		humanReadableDiskSize += "M"
+	}
+
 	diskSize, err := units.FromHumanSize(humanReadableDiskSize)
 	if err != nil {
+		fmt.Println("")
```

Comment:
Maybe we can also improve on the message for the memory flag:

     startFlagSet.Int(configCmd.Memory.Name, constants.DefaultMemory, "Amount of RAM to allocate to the Minishift VM.")

"Amount of RAM in MB to allocate to the Minishift VM. Also supports specification of unit MB, GB, TB." (there are probably better ways of expressing it). 

I am also wondering whether we should make the default "2048MB" instead of "2048", that would make the default self documenting.